### DISCUSSION TITLE
Excitement bump all rides when multiple colors used

### DISCUSSION THREAD

**Discussion by rondadams** [OpenRCT2/OpenRCT2#22017] [Feature requests]
It would be great if the excitement rating would increase when track/train colors are varied. 
For example, if building a coaster or car ride and the cars or trains have more than 1 color used, increase the excitement by 1-2 percent for each color. Same for track pieces.

Justification:
It takes a lot of extra clicks for a ride builder to change the colors, and it should add to the excitement as a reward for their efforts just like when they add other items to the surrounding area. 

Is there a way to make colors behave like the scenery objects?

**Comment by HtotheTML** [comment]
I'm not sure if this exact approach would be desirable for inciting players to use a lot of colors; what would instead make more sense is to broaden the eligibility of the Most Dazzling Ride Color Schemes award.

Presently you can only get this award if rides have their main color track set to specifically bright purple, bright green, light orange, or light pink (unless this has already been expanded). So there _is_ already existing utility for recoloring rides, but it could benefit from being broadened to more colors.

Most default ride color schemes already use multiple colors, so I don't know how good of a metric it would be to increase it on a per-color basis, especially if said ride has a lot of individual vehicles, like the bumper cars or Go-Karts. I can't say whether or not the number of clicks it takes to customize colors is enough of a justification to add that kind of bonus.

**Reply by rondadams** [reply]
Why shouldn’t it be any different than adding other scenery objects nearby?
I understand it might be difficult when an attraction already has multiple colors by default like go-karts, but getting higher excitement on those is somewhat easy. 
I’m mainly thinking for slow slow car rides and maybe coasters if multiple colors are done for the different trains or cars. 

Also, changing the name of an attraction from its default "X Ride 1" should have add to the excitement as well.

**Reply by SpartanFrederic104** [reply]
Now that you mentioned the most dazzling ride color schemes award, I already made a discussion about expanding the range of eligible colors for the most dazzling ride color schemes award quite a while back: #20161
